### ISSUE TITLE
Feature Request: Relative Path Imports

### ISSUE THREAD

**Issue by osterman** [cloudposse/atmos#293]
## Describe the Feature

Instead of requring the full path relative to the top-level, support relative paths to the current file. 

Today, we have to write:
```yaml
# orgs/acme/mgmt/auto/network.yaml
import:
  - orgs/acme/mgmt/auto/_defaults
```

Instead, we should be able to write:
```yaml
# orgs/acme/mgmt/auto/network.yaml
import:
  - _defaults
```

## Expected Behavior

Where by `_default` resolves to `orgs/acme/mgmt/auto/_default.yaml`; 

## Use Case

Make configurations easier to templatize without knowing the full path to the current working file.

## Describe Ideal Solution

Basically, take the  `dirname('orgs/acme/mgmt/auto/network.yaml')` and concatenate it with `/_default.yaml`

## Alternatives Considered

Explain what alternative solutions or features you've considered.

Today, we're using cookie-cutter when generating the files:

```
import:
  - orgs/{{ cookiecutter.namespace }}/mgmt/auto/_defaults
```
